lib/rmt/cli/import.rb

```diff
@@ -0,0 +1,28 @@
+class RMT::CLI::Import < RMT::CLI::Base
+
+  desc 'data PATH', 'Read SCC data from given path'
+  def data(path)
+    needs_path(path) do
+      RMT::SCC.new(options).import(path)
+    end
+  end
+
+  desc 'repos PATH', 'Mirror repos from given path'
+  def repos(path)
+    needs_path(path) do
+      repos = Repository.where(mirroring_enabled: true)
+      if repos.empty?
+        warn 'There are no repositories marked for mirroring.'
+        return
+      end
+
+      repos.each do |repo|
+        external_url = repo.external_url
+        repo.external_url = 'file://' + path + Repository.make_local_path(repo.external_url)
+        mirror(repo)
+        repo.update(external_url: external_url) # make sure we do not permanently change the URI
```

Comment:
It is! :ghost: And I don't like it myself, at all! I'll mark this comment with a fat `TODO` and see if I can workaround with `update_attribute`.

The background:
The repo resource gets updated later during mirroring to store the new timestamp. Other changes to the repo get saved along side. This line was a "hotfix" for that, but I guess I can refactor the problematic model methods to only update only one column.

Reply:
@ikapelyukhin @lagartoflojo 
I checked and this could be avoided by changing [refresh_timestamp!](https://github.com/SUSE/rmt/blob/master/app/models/repository.rb#L23) to only:
```ruby
  update_column(:last_mirrored_at, DateTime.now.utc)
```

But while this fixes the current issue, it masquerades the underlying problem too much, imo.
What do you think? Should I change it to that?
Or better add a `TODO` comment here and leave it for now?

Because I don't want to start another round of changes in this PR.

Reply:
@thutterer Maybe remove `refresh_timestamp!` method call from `RMT::Mirror` class and call it manually from wherever it makes sense to be called?

Reply:
Btw, you can just do: `touch(:last_mirrored_at)`